DABI.js

```diff
@@ -0,0 +1,677 @@
+{
+    "translatorID": "5cf8bb21-e350-444f-b9b4-f46d9fab7827",
+    "label": "DABI",
+    "creator": "Jens Mittelbach",
+    "target": "dabi.ib.hu-berlin\\.de/.*?",
+    "minVersion": "1.0",
+    "maxVersion": "",
+    "priority": 100,
+    "inRepository": true,
+    "translatorType": 4,
+    "browserSupport": "g",
+    "lastUpdated": "2015-12-15 18:59:56"
+}
+
+/* FW LINE 57:6869c32952b1 */
```

Comment:
Oh, nvm. Seems like you're not actually using the translator framework. Just remove the lines then.